packages/app-store/routing-forms/trpc/utils.ts

```diff
@@ -136,52 +163,86 @@ function getJsonLogic({
 
   const jsonLogicQuery = QbUtils.jsonLogicFormat(state.tree, state.config);
   const logic = jsonLogicQuery.logic;
-  if (!logic) {
-    if (attributesQueryValue.children1 && Object.keys(attributesQueryValue.children1).length > 0) {
-      throw new Error("Couldn't build the logic from the query value");
+
+  // We error only in previews to communicate any problem.
+  // In production, we don't error and instead prefer to let no members match which then causes all of the assignes of the team event to be used.
+  if (isPreview) {
+    const errors = getErrorsFromImmutableTree(state.tree).flat();
```

Comment:
Called only in case of preview so no impact on performance in live mode plus impact should be verify minimal like a few ms.